### ISSUE TITLE
DLAID and MOCKID are not integers

### ISSUE THREAD

**Issue by londumas** [igmhub/SaclayMocks#4]
DLAID and MOCKID are not unsigned integers (K) in the transmission files, instead they are floats.
This might prevent quickquasar from adding the DLA absorption.
```
import fitsio
h = fitsio.FITS('0/0/transmission-16-0.fits')
h[4].read_header()
h[4]['MOCKID'][:]
h[4]['DLAID'][:]
```

**Comment by TEtourneau** [comment]
I corrected this:


In [310]: dla[1]
Out[310]:

  file: /global/cscratch1/sd/tetourne/DesiMocks/v4.1.4/mock_0/output/master_DLA.fits
  extension: 1
  type: BINARY_TBL
  extname: DLACAT
  rows: 285022
  column info:
    RA                  f4
    DEC                 f4
    Z_QSO_NO_RSD        f4
    Z_QSO_RSD           f4
    Z_DLA_NO_RSD        f8
    Z_DLA_RSD           f8
    N_HI_DLA            f8
    MOCKID              i8
    DLAID               i8
    PIXNUM              i4

**Comment by londumas** [comment]
fixed in v4.2.
```
import fitsio
h = fitsio.FITS('0/0/transmission-16-0.fits.gz')
h[4].read_header()

XTENSION= 'BINTABLE'           / binary table extension
BITPIX  =                    8 / 8-bit bytes
NAXIS   =                    2 / 2-dimensional binary table
NAXIS1  =                   28 / width of table in bytes
NAXIS2  =                  221 / number of rows in table
PCOUNT  =                    0 / size of special data area
GCOUNT  =                    1 / one data group (required keyword)
TFIELDS =                    5 / number of fields in each row
TTYPE1  = 'Z_DLA_NO_RSD'       / label for field   1
TFORM1  = 'E       '           / data format of field: 4-byte REAL
TTYPE2  = 'Z_DLA_RSD'          / label for field   2
TFORM2  = 'E       '           / data format of field: 4-byte REAL
TTYPE3  = 'N_HI_DLA'           / label for field   3
TFORM3  = 'E       '           / data format of field: 4-byte REAL
TTYPE4  = 'MOCKID  '           / label for field   4
TFORM4  = 'K       '           / data format of field: 8-byte INTEGER
TTYPE5  = 'DLAID   '           / label for field   5
TFORM5  = 'K       '           / data format of field: 8-byte INTEGER
EXTNAME = 'DLA     '           / name of this binary table extension
```
